ql/src/test/queries/clientpositive/materialized_view_partitioned_create_rewrite_agg.q

```diff
@@ -0,0 +1,44 @@
+set hive.support.concurrency=true;
+set hive.txn.manager=org.apache.hadoop.hive.ql.lockmgr.DbTxnManager;
+
+CREATE TABLE t1(a int, b int,c int) STORED AS ORC TBLPROPERTIES ('transactional' = 'true');
+
+INSERT INTO t1(a, b, c) VALUES
+(1, 1, 1),
+(1, 1, 4),
+(2, 1, 2),
+(1, 2, 10),
+(2, 2, 11),
+(1, 3, 100),
+(null, 4, 200);
+
+CREATE MATERIALIZED VIEW mat1 PARTITIONED ON (a) STORED AS ORC TBLPROPERTIES ("transactional"="true", "transactional_properties"="insert_only") AS
```

Comment:
Can we add a couple of additional test similar to this MV?
i) Multiple partition columns.
ii) Multiple partition columns that do not match beginning/end of output columns, e.g., `SELECT a, b, c, d, e` and PARTITIONED BY `a, c, d`.

Reply:
added test for ii) since it also includes i)